codebundles/k8s-tail-logs-dynamic/.runwhen/generation-rules/k8s-tail-logs-dynamic.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: runwhen.com/v1
+kind: GenerationRules
+spec:
+  generationRules:
+    - resourceTypes:
+        - deployment
+      matchRules:
+        - type: and
+          matches:
+            - type: pattern
```

Comment:
I'm a bit worried about this match rule. Aside from ensuring it's been tested, "app" is a very common label, and I'm not sure we want a bunch of these just showing up on peoples maps if they aren't a fit. Do you think this useful across all app types? or should there be some other label that we also match on so that a user can "opt-in"?

Reply:
adding `codecollection.runwhen.com//app` as an annotation key to be added as an opt-in flag